src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -105,52 +94,21 @@ class Breakpoints extends Component<Props> {
   }
 
   renderBreakpoint(breakpoint) {
-    const locationId = breakpoint.locationId;
-    const line = breakpoint.location.line;
-    const column = breakpoint.location.column;
-    const isCurrentlyPaused = breakpoint.isCurrentlyPaused;
-    const isDisabled = breakpoint.disabled;
-    const isConditional = !!breakpoint.condition;
-    const isHidden = breakpoint.hidden;
-
-    if (isHidden) {
+    if (!breakpoint.id) {
```

Comment:
hmm, you can use `makeLocationId` to key the breakpoints... or keep the location id from the breakpoints map

this strips it out `breakpoints.valueSeq`